<!-- navigator_launch_file.launch -->
<launch>

<arg name="weights_file" default="$(find mattbot_image_detection)/weights/yolov8n-seg.pt"/>

  <!-- Launch the navigator.py script -->
  <node name="image_detector_node" pkg="mattbot_image_detection" type="detect_coco_with_dist_DDS.py" output="log">
    <param name="weights_file" value="$(arg weights_file)" />
  </node>

</launch>
